src/tree/scaling/ScaleItem.ts

```diff
@@ -54,7 +45,17 @@ export class ScaleItem implements RevisionsItemModel, RevisionsDraftModel {
     }
 
     private get parentResource(): ContainerAppModel | Revision {
```

Comment:
Since this is private, is there any real reason to not just call `getParentResource` directly from the util?

Reply:
Nah not really, mostly just calling it it through a getter looks a little cleaner, but functionally adds nothing else

Reply:
`this.parentResource` vs `getParentResource(this.containerApp, this.revision)` each time

Reply:
Yeah, that's a good point. I agree.